src/Model/Page.php

```diff
@@ -582,6 +589,22 @@ public function setTarget(PageInterface $target = null): void
         $this->target = $target;
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
can you remove this comment?

Reply:
It's the same on all the other getters / setters in this file which is why i set it there too, so it's consistent. But sure, I can remove it.

Reply:
Recently there was an agreement that we should not use `{@inheritdoc}` anymore, but no one went through all the code and removed it everywhere 😅